### ISSUE TITLE
LHR lock 

### ISSUE THREAD

**Issue by cor8** [NebuTech/NBMiner#622]
After LHR lock is detected 2 times in short time it drops from LHR 74 to LHR 73.5. Can I change this that it drops only by 0.1 instead by 0.5?

**Comment by igralec84** [comment]
Yes you can also set that it doesn't drop at all: -lhr-reduce-value 0 -lhr-reduce-limit 1

**Comment by MdqMiner** [comment]
> Sí, también puede configurar que no se caiga en absoluto:
> 
> `-lhr-reduce-value 0 -lhr-reduce-limit 1`

HI, im begginer. how change this value? where? thanks.

**Comment by igralec84** [comment]
> > Sí, también puede configurar que no se caiga en absoluto:
> > `-lhr-reduce-value 0 -lhr-reduce-limit 1`
> 
> HI, im begginer. how change this value? where? thanks.

Just edit your .bat file that starts the miner to something like this:

`nbminer -a ethash -o stratum+ssl://eu1.ethermine.org:5555 -u WALLET.WORKER -lhr-reduce-value 0 -lhr-reduce-limit 1`

**Comment by MdqMiner** [comment]
ohh thanks so much. its so helpful to me!!
